gobblin-core/src/main/java/gobblin/source/extractor/extract/kafka/ConfluentKafkaSchemaRegistry.java

```diff
@@ -74,7 +79,8 @@ protected Schema fetchSchemaByKey(Integer key) throws SchemaRegistryException {
   @Override
   public Schema getLatestSchemaByTopic(String topic) throws SchemaRegistryException {
     try {
-      return new Schema.Parser().parse(this.schemaRegistryClient.getLatestSchemaMetadata(topic).getSchema());
+      String schemaName = topic + props.getProperty(CONFLUENT_SCHEMA_NAME_SUFFIX, CONFLUENT_SCHEMA_DEFAULT_NAME_SUFFIX);
```

Comment:
Replace the props.getProperty invocation with a member variable: schemaNameSuffix which is populated during construction of the schema registry from the passed in props. 

String schemaName = topic + schemaNameSuffix;